doc/FAQ.md

```diff
@@ -34,13 +34,21 @@ distributed systems. I just wanted the fastest way to boot to Docker.
 
 **Persistent partition choice**
 
-Boot2Docker will first try to mount a partition labeled ``boot2docker-data``, if
+Boot2Docker will first try to mount a partition named ``boot2docker-data``, if
 that doesn't exist, it will pick the first ``ext4`` partition listed by ``blkid``.
 
+By default, the ISO image creates the ``boot2docker-vm`` partition, therefore no additional one is likely to be needed but you can still add some if you want to.
+
+Keep in mind that if you have 2 partitions with the same names but different labels,
+boot2docker will only see one of them. So, if you add a ``boot2docker-data`` partition, boot2docker will mount that one and forget ``boot2docker-vm``.
+However, if you name them differently, boot2docker will see all of them.
+E.g.: ``boot2docker-vm`` and ``partition2`` will both be mounted.
```

Comment:
You're right. I can't find back the issue I was inspired from, they were telling something like I did. I did some tests and the only partition that seems to be mounted is `boot2docker-vm`.

Anyway, to fix my commit I need some more information. According to the script you linked, `boot2docker-vm` shouldn't even be loaded BUT according to my tests no partition can be loaded except `boot2docker-vm`. Even `boot2docker-data` isn't loaded if attached alone. Maybe there's a link with #339 but I didn't check much.